conf/scalardb-custom-values.yaml

```diff
@@ -14,42 +14,45 @@ envoy:
     type: LoadBalancer
     annotations:
       service.beta.kubernetes.io/azure-load-balancer-internal: "true"
-      service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "k8s_ingress"
```

Comment:
The current manual deployment guide uses another (separated from the worker node) subnet for the deployment of the Load Balancer.
However, it is not required. And, I was not able to find a clear reason why the current document separates the subnet for Load Balancer.
As a result of my testing, it works properly even if I don't separate the subnet for Load Balancer.
So, I removed this configuration (specify the other subnet) in this sample custom values file.